.github/workflows/kernel_integration.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Kernel Integration Tests'
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  build-initramfs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: build initramfs
+        run: |
+          go install github.com/florianl/bluebox@latest
```

Comment:
Let's use a pinned commit to avoid unexpected behavior. We'll have to update it from time to time but I think it's worth it.

I'd prefer to not use external tools though. Why is bluebox required? What about using a base file (e.g. from ubuntu-latest) and append the required test files to the cpio archive?